test/java_time/dev/gen.clj

```diff
@@ -54,11 +64,10 @@
             (throw (IllegalArgumentException.
                      (str "Calling import-macro on a non-macro: " sym))))
         n (:name m)
-        arglists (:arglists m)]
-    (list* 'defmacro n 
+        arglists (:arglists m)
+        forwarded-meta (not-empty (into (sorted-map) (select-keys m [:doc :deprecated])))]
+    (list* 'defmacro (with-meta n forwarded-meta)
```

Comment:
This was necessary, because `defmacro` docstrings were still printed the old way (with `\n`).  With this change, they undergo the same `DocString` treatment as `defn` and `ns`.